src/operator/nn/fully_connected.cc

```diff
@@ -165,6 +165,7 @@ static bool FullyConnectedType(const nnvm::NodeAttrs& attrs,
       attrs, in_type, out_type, -1);
 }
 
+
```

Comment:
why new line?

Reply:
style guide says it's ok 1 or 2 blank lines around functions, more consistent with python as we use two blank lines between functions according to pep8.

Reply:
I know pep8 has this rule for python, but does it also apply to C++?

Reply:
I think is nice, and google style allows up to two newlines, so I don't know why are we discussing this. If it makes you so unhappy, I remove the newline. For me it  helps readability.